util/topsql/tracecpu/profile.go

```diff
@@ -43,15 +43,19 @@ const (
 // GlobalSQLCPUProfiler is the global SQL stats profiler.
 var GlobalSQLCPUProfiler = newSQLCPUProfiler()
 
-// Collector uses to collect SQL execution cpu time.
-type Collector interface {
+// Reporter uses to collect SQL execution cpu time.
```

Comment:
Prefer to keep use the original `Collector` name.
`Reporter` should contain `Collector` interface, but `tracecpu` doesn't care about `repotor`, it only knows the collector.

Reply:
Fixed.